### DISCUSSION TITLE
1.8.3

### DISCUSSION THREAD

**Discussion by 3arthqu4ke** [3arthqu4ke/3arthh4ck#242] [Announcements]
A bunch of QoL improvements and some extremly specific settings.
* Sorter, bugfixes, EnsureHotbarSorting
* Offhand - SwordGapCrystal, fixed CrystalCheck (redundant anyways)
* AutoMine - DamageCheck, ExtraBurrowCheck and PrioSelfWithStep
* Speedmine -
    *  Place/BreakCrystal
    * some workarounds to SilentSwitch
    * compatibility with AutoTrap
* AutoArmor - maybe improved desync from AutoMending?
* AutoCrystal - 
    * BasePlaceOnly
    * SafetyFactor
    * Fixed random breaking by applying MinBreakDamage
    * SetDead improvements, CountDead
    * PlaceRangeEyes
    * Some Extrapolation settings
* Replenish - InInvWithMiddleClick
* ExpTweaks - SilentOnlyWhenUsing
* Speed - Mode None and Modifier
* AccountSpoof - SpoofSinglePlayer, Media support, RandomUUID
* AntiSurround - StopOnObby
* Surround - Movement - Y and Yplus
* Updated eucc config on the [configs](https://github.com/3arthqu4ke/3arthh4ck/tree/configs) branch.
* [BackupPlugin](https://github.com/3arthqu4ke/3arthh4ck/actions/runs/3011000053)
@3arthqu4ke 

<hr /><em>This discussion was created from the release <a href='https://github.com/3arthqu4ke/3arthh4ck/releases/tag/1.8.3'>1.8.3</a>.</em>

**Comment by 3arthqu4ke** [comment]
Hi

**Reply by master7720** [reply]
Hi(sorry sir for my poor english)

**Reply by Zane2b2t** [reply]
J

**Comment by Zane2b2t** [comment]
@3arthqu4ke add on discord :pray:

I have a notepad full of stuff that'd be too spammy for gh issues
ZANE#4417

(Or i could post them 1 by 1 for more orginzation)

**Reply by Zane2b2t** [reply]
i sent so much issues in 30 minutes github thinks i sent them 13 hours ago...
